tabs/motors.js

```diff
@@ -477,6 +479,8 @@ TABS.motors.initialize = function (callback) {
             }
 
             $('div.sliders input').trigger('input');
+
+            mspHelper.setArmingDisabled(!enabled);
```

Comment:
Double negative makes the code harder to understand. When checkbox is enabled then arming is enabled also. Had to read it several times to wrap my head around it.